publish.sh

```diff
@@ -88,12 +85,32 @@ if [ "$PROJECT" == "orm" ] || [ "$PROJECT" == "reactive" ]; then
 	  EXTRA_ARGS=" --dry-run"
 	fi
 
-  ./gradlew releasePerform closeAndReleaseSonatypeStagingRepository -x test \
-    --no-scan --no-daemon --no-build-cache --stacktrace $EXTRA_ARGS \
-    -PreleaseVersion=$RELEASE_VERSION -PdevelopmentVersion=$DEVELOPMENT_VERSION \
-    -PdocPublishBranch=production -PgitRemote=origin -PgitBranch=$BRANCH
+	if [ -f "./jreleaser.yml" ]; then
+		# JReleaser-based build
+		export JRELEASER_GPG_HOMEDIR="$RELEASE_GPG_HOMEDIR"
+		# Get the jreleaser downloader
+		curl -sL https://git.io/get-jreleaser > get_jreleaser.java
```

Comment:
That, uh... is very trusting of you.
Is this file expected to change? If not, can we bundle it within our scripts instead, and only update it when necessary?

Reply:
> ideally ... I'd prefer if we could just `sdk install jreleaser` as part of the release image and don't bother with this download stuff in the scripts ...

There's a package on Fedora COPR, apparently. Also a flatpak. We can definitely update our CI images to make sure jreleaser is installed there.

IMO either we do that, or we reduce the amount of things we download here to the _bare minimum_. Each website we download things from (`git.io`, `jreleaser.org`, ...) during releases is another way to get pwned.

AFAICS, the bare minimum would be something like `curl https://github.com/jreleaser/jreleaser/releases/download/v1.17.0/jreleaser-1.17.0.zip ; unzip ...`. Which looks reasonable enough.